spotinst/ocean_aws_logging/fields_spotinst_ocean_aws_launch_spec.go

```diff
@@ -0,0 +1,160 @@
+package ocean_aws_logging
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/spotinst/spotinst-sdk-go/service/ocean/providers/aws"
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+	"github.com/spotinst/terraform-provider-spotinst/spotinst/commons"
+)
+
+func Setup(fieldsMap map[commons.FieldName]*commons.GenericField) {
+
+	fieldsMap[Logging] = commons.NewGenericField(
+		commons.OceanAwsLogging,
+		Logging,
+		&schema.Schema{
+			Type:     schema.TypeSet,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					string(Export): {
+						Type:     schema.TypeSet,
```

Comment:
Please change to `TypeList` since  `Export ` is a is a single object and set of objects.